### PR TITLE
Revert "Protect staging from 'Update functional tests for buyer-frontend#812'"

### PR DESCRIPTION
**Pull request by lfdebrux**
We are releasing alphagov/digitalmarketplace-buyer-frontend#812 to
staging.

This reverts commit d42087350f4f1532c82b1e0a6744d378e80726f7.

### CODE REVIEW

File: features/smoke-tests/buyer/catalogue.feature [hunk 1]
```diff
@@ -15,14 +15,14 @@ Scenario: User can see the main links on the homepage
 
 @skip-local
 @skip-preview
+@skip-staging
```
Comment (Wynndow) [inline features/smoke-tests/buyer/catalogue.feature]: If this scenario is being skipped everywhere, can it be removed?

Comment (lfdebrux) [reply features/smoke-tests/buyer/catalogue.feature]: It will be in the next revert

Comment (lfdebrux) [reply features/smoke-tests/buyer/catalogue.feature]: It's still being run on production

Comment (Wynndow) [reply features/smoke-tests/buyer/catalogue.feature]: Oh wait, smoke tests....my bad,

**Pull request merged**